components/elm/bld/namelist_files/namelist_definition.xml

```diff
@@ -2020,5 +2020,15 @@ explain the SZA dependence of direct irradiance.
 Toggle to turn on the internal mixing of dust-snow. 
 </entry>
 
+<!-- ========================================================================================  -->
+<!-- Namelist option for MPI syncing                                                           -->
+<!-- ========================================================================================  -->
+
+<entry id="mpi_sync_nstep_freq" type="integer(6)" category="history"
+       group="elm_inparm" value="0">
+The number of steps after which a MPI_Barrier is called. By default (-1), a MPI_Barrier will
```

Comment:
Where are you setting the default of `-1`?

Reply:
I think the logic you have works for >0 and the default is set to zero. So probably just update the comment.

Reply:
I have updated the comment.